src/Pmi/Controller/SymfonyMigrationController.php

```diff
@@ -121,6 +125,24 @@ public function orderAction($participantId, $orderId, Application $app)
         ));
     }
 
+    /**
+     * @deprecated 2021-05-10
+     */
+    public function evaluationAction($participantId, $evalId, Application $app)
+    {
+        if (!$evalId) {
+            return $app->redirect(sprintf(
+                '/s/participant/%s/measurements',
```

Comment:
Will we need trailing slash(es)?

Reply:
I don't think so since it's not an index route `/` like workqueue